packages/migrate/src/types.ts

```diff
@@ -322,6 +370,14 @@ export namespace EngineArgs {
     affected: AffectedModelOrCompositeTypeAndField[]
   }
 
+  export type ViewWarningCodes =
```

Comment:
Unless I am missing something, this type is used only for test, so I would've moved it there.

Reply:
The reasoning for this was that the underlying types _aren't_ exported but I'm not sure if that's something that we can just update or if we should leave as is

Reply:
Ah, ok then, I think in that case it's fine to keep it where it is.